client/src/components/settings/toggle-radio-setting.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import '../helpers/toggle-button.css';
+import './toggle-setting.css';
+
+type ToggleRadioSettingProps = {
+  action: string;
+  explain?: string;
+  flag: boolean;
+  flagName: string;
+  toggleFlag: () => void;
+  offLabel: string;
+  onLabel: string;
+};
+
+export default function ToggleRadioSetting({
+  action,
+  explain,
+  flag,
+  flagName,
+  toggleFlag,
+  ...restProps
+}: ToggleRadioSettingProps): JSX.Element {
+  const randomId = Math.floor(Math.random() * 1_000_000);
+  const firstRadioId = `radioA${randomId}`;
+  const secondRadioId = `radioB${randomId}`;
```

Comment:
Same as toggle button
```suggestion
```

Reply:
I think it is best if we leave the `firstRadioId` and `secondRadioId` declarations here (and just use `flagName` instead of `randomId`) since these values are each used in two places below and thus if changed them for some reason we would only need to change them once here.